modules/custom/az_paragraphs/config/optional/editor.editor.az_standard.yml

```diff
@@ -4,96 +4,277 @@ dependencies:
+          element: '<span class="lead">'
+        -
+          label: 'Text Size H1'
+          element: '<span class="h1">'
+        -
+          label: 'Text Size H2'
+          element: '<span class="h2">'
+        -
+          label: 'Text Size H3'
+          element: '<span class="h3">'
+        -
+          label: 'Text Size H4'
+          element: '<span class="h4">'
+        -
+          label: 'Text Size H5'
+          element: '<span class="h5">'
+        -
+          label: 'Text Size H6'
+          element: '<span class="h6">'
+        -
+          label: 'Text Align Top'
+          element: '<span class="align-top">'
+        -
+          label: 'Text Align Middle'
+          element: '<span class="align-middle">'
+        -
+          label: 'Text Align Bottom'
+          element: '<span class="align-bottom">'
+        -
+          label: 'Zero Margin Top'
+          element: '<span class="mt-0">'
+        -
+          label: 'Zero Margin Bottom'
+          element: '<span class="mb-0">'
+        -
+          label: Blockquote
+          element: '<blockquote class="blockquote">'
+        -
+          label: 'Blockquote Center'
+          element: '<blockquote class="blockquote text-center">'
+        -
+          label: 'Blockquote Right'
+          element: '<blockquote class="blockquote text-right">'
+        -
+          label: 'Blockquote Reverse'
+          element: '<blockquote class="blockquote blockquote-reverse text-right">'
+        -
+          label: 'Blockquote Footer'
+          element: '<footer class="blockquote-footer">'
+        -
+          label: 'Bordered table'
+          element: '<table class="table table-bordered">'
+        -
+          label: 'Hover table'
+          element: '<table class="table table-hover">'
+        -
+          label: 'Small table'
+          element: '<table class="table table-sm">'
+        -
+          label: 'Striped table'
+          element: '<table class="table table-striped">'
+        -
+          label: 'Dark table'
+          element: '<table class="table table-dark">'
+        -
+          label: 'Dark table heading'
+          element: '<thead class="thead-dark">'
+    ckeditor_bs_grid_grid:
       use_cdn: false
       cdn_url: ''
       available_columns:
-        1: '1'
-        2: '2'
-        3: '3'
-        4: '4'
-        5: '5'
-        6: '6'
-        7: '7'
-        8: '8'
-        9: '9'
-        10: '10'
-        11: '11'
-        12: '12'
+        - '1'
+        - '2'
+        - '3'
+        - '4'
+        - '5'
+        - '6'
+        - '7'
+        - '8'
+        - '9'
+        - '10'
+        - '11'
+        - '12'
       available_breakpoints:
-        xs: xs
-        sm: sm
-        md: md
-        lg: lg
-        xl: 0
-        xxl: 0
+        - xs
+        - sm
+        - md
+        - lg
+        - xl
+    linkit_extension:
+      linkit_enabled: true
+      linkit_profile: az_linkit
+    media_media:
+      allow_view_mode_override: true
```

Comment:
What is this `allow_view_mode_override` setting?

Reply:
Not sure.  Was automatically added by the editor config upgrade.